applications/lodex-workers-python/config.json

```diff
@@ -13,11 +13,10 @@
 		"PIP_NO_CACHE_DIR": 0
 	},
 	"packages": [
-		"@ezs/spawn@1.1.0",
-		"@ezs/analytics@1.18.4",
-		"@ezs/basics@1.17.1"
+		"@ezs/spawn@1.2.31",
```

Comment:
if the packages are the same as in the package.json file, I think it is not necessary to specify them here

Reply:
Nevertheless it has an advantage: when admins wants to update the packages, their names are already present in the configuration.